applications/dashboard/openapi/schemas.yml

```diff
@@ -3,6 +3,18 @@ info:
 paths:
 components:
   schemas:
+    Format:
+      description: The format of the body used to convert it to HTML.
+      type: string
+      enum:
+        - rich
+        - markdown
+        - text
+        - textex
+        - wysiwyg
+        - bbcode
+        - display
```

Comment:
You'd documented `display` here, but it's not actually in the main schema.

Additionally until we actually create some `DisplayFormat` and register it with the format service, that format would display errors in a bunch of places (search, activity, notifications, etc).